preprocessors/yolov8/detect.py

```diff
@@ -15,364 +15,207 @@
 # If not, see
 # <https://github.com/Shared-Reality-Lab/IMAGE-server/blob/main/LICENSE>.
 
-
-import time
-from pathlib import Path
-from flask import Flask, jsonify, request
-import cv2
-import torch
-import base64
-import numpy as np
-import jsonschema
+from datetime import datetime
 import json
+import time
 import logging
 import os
+import traceback
+from flask import Flask, request, jsonify
+import jsonschema
+import base64
+import io
+from PIL import Image
+from ultralytics import YOLO
+import torch
 
-from datetime import datetime
-from ultralytics.nn.tasks import attempt_load_weights
-from ultralytics.yolo.utils import plt_settings
-from ultralytics.yolo.utils.torch_utils import select_device
-from ultralytics.yolo.utils.checks import check_imgsz
-from ultralytics.yolo.utils.ops import scale_coords, non_max_suppression
-
-os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
+# Create Flask app
 app = Flask(__name__)
+
+# Configure logging
 logging.basicConfig(level=logging.DEBUG)
+log_pii = True
+
+# Environment variables and constants
+MODEL_PATH = os.environ.get(
+    'YOLO_MODEL_PATH',
+    'https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11x.pt'
+    )
+CONF_THRESHOLD = float(os.environ.get('CONF_THRESHOLD', '0.75'))
```

Comment:
These parameters should be logged, so we know from logs, for example, the confidence threshold for debugging.

Reply:
(logging added)